components/playlist/browser/playlist_thumbnail_downloader.cc

```diff
@@ -122,15 +117,14 @@ void PlaylistThumbnailDownloader::SaveResponseToFile(
   }
 
   if (!response_body) {
-    url_loaders_.erase(iter);
```

Comment:
Maybe this point causes UaF failure?
IIUC, `url_loaders_` should have `iter` if `url_loader_map_` has `id`. isn't it?

Reply:
Yeah, that was what I expected, but it seems it didn't help 🥲

Reply:
Maybe list iterator could be invalidated after its container is changed. I assumed that.